app/src/main/java/org/thoughtcrime/securesms/ui/theme/MaybeFollowSystemColors.kt

```diff
@@ -0,0 +1,20 @@
+package org.thoughtcrime.securesms.ui.theme
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.runtime.Composable
+
+fun interface MaybeFollowSystemColors {
```

Comment:
I don't think we should have classes with `Maybe` in them. If this is an interface that provides something then it should probably be called something along the lines of `ThemeColorsProvider`

Reply:
I like it